index.html

```diff
@@ -23,93 +23,105 @@
   <div>
     <img src="img/emacs.png" alt="Emacs" class="right"/>
     <p>
-      Emacs is a <strong>text editor</strong> which you can customize
-      and extend in so many ways. It was written in C and provides its
-      own implementation of <strong>Lisp</strong>,
-      <a href="https://xkcd.com/224/"> the language from which <em>the
-      Gods wrought the universe</em></a>.
+      Emacs is a <strong>highly extensible text editor</strong>. It is
+      built on a C core, and provides a fully
+      featured <strong>Lisp</strong>environment
+      (<a href="https://xkcd.com/224/">the language from which <em>the
+      Gods wrought the universe</em></a>).
     </p>
     <p>
       There's a reason why Emacs was
-      <a href="http://jtnimoy.net/workviewer.php?q=178" title="Emacs on
-      Tron Legacy">featured in Tron Legacy</a>, it is sexy! (and
-      awesome).
+      <a href="http://jtnimoy.net/workviewer.php?q=178" title="Emacs
+      on Tron Legacy">featured in Tron Legacy</a> - because it's sexy!
     </p>
     <p>
-      You can use Emacs for all of your text processing
-      needs, internet and social network interactions, hacking,
-      coding, managing to-do's and organizing your daily schedule,
-      play Tetris and it can even provide you with free psychotherapy and
-      counseling using <code>M-x doctor</code>.
+      You can use Emacs for all of your text processing needs,
+      internet and social network interactions, hacking, coding,
+      managing to-do's and organizing your daily schedule, playing
+      Tetris, and many other awesome things. It can even provide you
+      with free psychotherapy and counseling using the built
+      in <code>doctor</code>. A vibrant community of hackers are
```

Comment:
Community is singular, so it should use `is` or just `hackers are`.  I'd go with `is`.